### PR TITLE
fix: add parsing for `visible`, `attributes`, and `properties`

### PR DESCRIPTION
**Pull request by jrandolf**
This PR adds parsing to the attributes added in https://github.com/puppeteer/replay/pull/412.

### CODE REVIEW

**Review by OrKoN — CHANGES_REQUESTED** [review]
please add tests for parsing

**Comment by jrandolf** [comment]
> please add tests for parsing

Done

File: test/SchemaUtils.test.ts [hunk 1]
```diff
@@ -322,6 +322,9 @@ describe('SchemaUtils', () => {
               selectors: [['aria/Test']],
               operator: '==',
               count: 1,
+              properties: {},
```
Comment (OrKoN) [inline test/SchemaUtils.test.ts:325]: let's add tests with non empty attributes too. And a test when properties or attributes are of invalid types,

File: src/SchemaUtils.ts [hunk 2]
```diff
@@ -429,6 +436,13 @@ function parseWaitForElementStep(step: object): WaitForElementStep {
     type: StepType.WaitForElement,
     operator: operator as '>=' | '==' | '<=' | undefined,
     count: parseOptionalNumber(step, 'count'),
+    visible: parseOptionalBoolean(step, 'visible'),
+    attributes: hasProperty(step, 'attributes')
+      ? (step.attributes as { [name: string]: string })
+      : undefined,
+    properties: hasProperty(step, 'properties')
+      ? (step.properties as { [name: string]: string })
```
Comment (OrKoN) [inline src/SchemaUtils.ts:444]:
```suggestion
      ? (step.properties as { [name: string]: unknown })
```

**Review by OrKoN — APPROVED** [review]

File: test/SchemaUtils.test.ts (hunk 1 above)
Comment (SaltonSea374) [reply test/SchemaUtils.test.ts:325]: What a help

**Review by SaltonSea374 — COMMENTED** [review]
Done
